Mutagen.Bethesda.Analyzers.SDK/Topics/ITopic.cs

```diff
@@ -5,4 +5,5 @@ public interface ITopic
     TopicDefinition TopicDefinition { get; }
     string FormattedMessage { get; }
     Severity Severity { get; set; }
+    IEnumerable<object?> Items { get; }
```

Comment:
Don't know if I'm on board of disrupting the interface like this yet.   I'd want to think about the EditorID enrichment card in isolation a bit to know if this was worth it or not... but I'd rather not hold up this PR for it.   We can always mix in EditorID enrichment later.. it's not a blocker for any of the rest of the content I dont think

Can we back out the changes for this card for now?
- This `Items` field
- FuncRecordTopic class
- Topic enrichers

Reply:
I personally think this is quite essential to have analyzers that you can properly work with. I refactored a lot of code so we pass form links which makes the development experience a lot better when you don't have to care about getting any editor ids. If you think there is a good way to handle this differently let me know! I agree it's a bit hacky right now. Maybe we could add a new type which combines something like `FormattedMessage` and `Items` which are both concerned with the resulting message sent to the dropbox. I personally wouldn't mind waiting a bit with the PR while we're figuring this out. Feels incomplete otherwise.